src/contract_wrappers/tokens/__tests__/security_token_wrapper.test.ts

```diff
@@ -0,0 +1,2862 @@
+    });
+  });
+
+  describe('featureRegistry', () => {
+    test('should call to featureRegistry', async () => {
+      const expectedResult = 'string';
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.featureRegistry).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync()).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.featureRegistry();
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.featureRegistry).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('securityTokenRegistry', () => {
+    test('should call to securityTokenRegistry', async () => {
+      const expectedResult = 'string';
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.securityTokenRegistry).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync()).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.securityTokenRegistry();
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.securityTokenRegistry).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('tokenDetails', () => {
+    test('should call to tokenDetails', async () => {
+      const expectedResult = 'string';
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.tokenDetails).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync()).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.tokenDetails();
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.tokenDetails).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('transfersFrozen', () => {
+    test('should call to transfersFrozen', async () => {
+      const expectedResult = true;
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.transfersFrozen).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync()).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.transfersFrozen();
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.transfersFrozen).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('getModulesByName', () => {
+    test('should call to getModulesByName', async () => {
+      const expectedResult = true;
```

Comment:
getModulesByName should not return a boolean. It should be string[].
These kind of errors will raise once we start writing return types for every method.